qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -85,7 +95,7 @@ def __init__(
         values in the loaded account.
 
         Args:
-            channel: Channel type. ``ibm_cloud`` or ``ibm_quantum``.
+            channel: Channel type. ``ibm_cloud``, ``ibm_quantum`` or ``local``.
```

Comment:
Can we add a little more information on `local` - e.g. 

If `local` is specified, the local testing mode is used, and primitive queries will run on a local simulator. See [Qiskit Runtime local testing mode](https://docs.quantum.ibm.com/guides/local-testing-mode) for more details.